ingestion-sink/src/test/resources/testdata/payload-format-expected.ndjson

```diff
@@ -23,3 +23,5 @@
 {"additional_properties":{},"test_nested_list":[{"list":["a","b","c"]}],"test_nested_csv":[{"list":"a,b,c"}],"test_list":["a","b","c"]}
 {"additional_properties":{},"test_nested_list":[{}]}
 {"additional_properties":{}}
+{"additional_properties":{"test_int64_":"invalid"},"test_int64":8}
```

Comment:
This result isn't making sense to me. Does the `8` value win because it's processed last?

Reply:
> This result isn't making sense to me.

This test case intentionally highlights behavior that may be unexpected. I've filed #1298 to follow up on this.

> Does the `8` value win because it's processed last?

yes, because both fields map to `test_int64` and contain valid data, both fields are written to there, with the later values writing over earlier values.